phoenix/core/migrations/0032_auto_20190827_0728.py

```diff
@@ -0,0 +1,111 @@
+# Generated by Django 2.2 on 2019-08-27 07:28
+
+from django.db import migrations, models
+
+
+def map_eta(eta):
+    if eta < 30:
+        new_eta = "<30m"
+    elif eta < 120:
+        new_eta = "<2h"
+    elif eta < 480:
+        new_eta = "<8h"
+    elif eta < 1440:
+        new_eta = "<24h"
+    else:
+        new_eta = ">24h"
+    return new_eta
+
+
+def transform_previous_eta_values(apps, schema_editor):
+    Outage = apps.get_model("core", "Outage")
+    OutageHistory = apps.get_model("core", "OutageHistory")
+    for outage in Outage.objects.all():
+        eta = int(outage.eta)
+        outage.eta = map_eta(eta)
+        outage.save()
+        for outage_history in OutageHistory.objects.filter(outage_id=outage.id):
```

Comment:
I suppose that it could be a single query instead of 1 query per outage, seems that extra filtering is not necessary here - we will update all outage history entries with not null `outage_id`

Reply:
You're right, changed. I copied boilerplate code from previous migration file and didn't think about it.